pkg/monitortests/node/watchnodes/node.go

```diff
@@ -237,3 +274,27 @@ func findNodeCondition(status []corev1.NodeCondition, name corev1.NodeConditionT
 	}
 	return nil
 }
+
+func isNodeReady(node *corev1.Node) bool {
+	isReady := false
+	if node == nil {
+		return isReady
+	}
+	if c := findNodeCondition(node.Status.Conditions, corev1.NodeReady, 0); c != nil {
+		isReady = c.Status == corev1.ConditionTrue
+	}
+	return isReady
+}
+
+func doesNodeHaveUnreachableTaints(node *corev1.Node) bool {
+	if node == nil {
+		return false
+	}
+	annotations := node.Annotations
```

Comment:
why not `node.spec.taints`?

Reply:
you are right. Pushed up a fix for that.